apps/test-can/Arm/main.cpp

```diff
@@ -0,0 +1,72 @@
+#include "mbed.h"
+#include "CANMsg.h"
+
+CAN                 can(CAN_RX, CAN_TX);
+CANMsg              rxMsg;
+CANMsg              txMsg;
+DigitalOut          ledTX(LED1);
+DigitalOut          ledRX(LED2);
+Timer               timer;
+uint8_t             counter = 0;
+
+#define TX_ID 0x100
+#define RX_ID 0x101
+ 
+/**
+ * @brief   Prints CAN msg to PC's serial terminal
+ * @note
+ * @param   CANMessage to print
+ * @retval
+ */
+void printMsg(CANMessage& msg) {
+    printf("  ID      = 0x%.3x\r\n", msg.id);
```

Comment:
What's the point of having \r before each \n?

Reply:
This is so serial output still works in windows. Windows represents newlines as Carriage Return + Line Feed, whereas UNIX-like systems use just a Line Feed

Reply:
Screen for serial monitoring even on unix also requires return + line feed